test/unit_test/distributionally_robust_test.cc

```diff
@@ -50,11 +50,16 @@ BOOST_AUTO_TEST_CASE(distributionally_robust_recompute_duals)
       {false, 0.5602916549924593, -0.9880343238436395, 0, 8.964083874125915},
       {false, 0.5684515391242324, -1.0040272155608332, 0, 9.95511979025179}};
 
+  float l_bound[] = {0, 0.09662899139580064, 0.04094541671274077, 0.12209962713632538, 0.1418901366388003,
```

Comment:
How is this calculated?

Reply:
see line 33 of that same file, it is generated by the python script also changed in this same diff